Generated application typescript build is broken

### Expected behavior
Generated application should be able to build as it is

### Actual behavior
```

src/commands/transaction/create.ts:7:9 - error TS2742: The inferred type of 'flags' cannot be named without a reference to '../../../../../10_lisk/lisk-sdk/node_modules/@oclif/command/lib/flags'. This is likely not portable. A type annotation is necessary.

7  static flags = {
          ~~~~~

src/commands/transaction/create.ts:7:9 - error TS2742: The inferred type of 'flags' cannot be named without a reference to 'lisk-commander/node_modules/@oclif/parser/lib/flags'. This is likely not portable. A type annotation is necessary.

7  static flags = {
          ~~~~~

src/commands/transaction/sign.ts:7:9 - error TS2742: The inferred type of 'flags' cannot be named without a reference to '../../../../../10_lisk/lisk-sdk/node_modules/@oclif/command/lib/flags'. This is likely not portable. A type annotation is necessary.

7  static flags = {
          ~~~~~

src/commands/transaction/sign.ts:7:9 - error TS2742: The inferred type of 'flags' cannot be named without a reference to 'lisk-commander/node_modules/@oclif/parser/lib/flags'. This is likely not portable. A type annotation is necessary.

7  static flags = {
          ~~~~~

```

### Steps to reproduce
From commander,
./bin/run init path
cd path
npm run build

### Which version(s) does this affect? (Environment, OS, etc...)
feature//6066-commander-bootstraping

--- a/commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/app/modules.ts
+++ b/commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/app/modules.ts
@@ -1,3 +1,4 @@
 import { Application } from 'lisk-sdk';
 
+// eslint-disable-next-line @typescript-eslint/no-empty-function
 export const registerModules = (_app: Application): void => {};

--- a/commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/genesis-block/create.ts
+++ b/commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/genesis-block/create.ts
@@ -4,6 +4,7 @@ import { getApplication } from '../../app/app';
 import { registerModules } from '../../app/modules';
 
 export class GenesisBlockCommand extends BaseGenesisBlockCommand {
+	// eslint-disable-next-line class-methods-use-this
 	public getApplication(
 		genesisBlock: Record<string, unknown>,
 		config: PartialApplicationConfig,

--- a/commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/start.ts
+++ b/commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/start.ts
@@ -51,8 +51,10 @@ const setPluginConfig = (config: ApplicationConfig, flags: Flags): void => {
 	}
 };
 
+type StartFlags = typeof BaseStartCommand.flags & { [key: string]: Record<string, unknown> };
+
 export class StartCommand extends BaseStartCommand {
-	static flags: any = {
+	static flags: StartFlags = {
 		...BaseStartCommand.flags,
 		'enable-http-api-plugin': flagParser.boolean({
 			description:

--- a/commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/transaction/create.ts
+++ b/commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/transaction/create.ts
@@ -3,8 +3,12 @@ import { TransactionCreateCommand } from 'lisk-commander';
 import { Application, PartialApplicationConfig } from 'lisk-sdk';
 import { getApplication } from '../../app/app';
 
+type CreateFlags = typeof TransactionCreateCommand.flags & {
+	[key: string]: Record<string, unknown>;
+};
+
 export class CreateCommand extends TransactionCreateCommand {
-	static flags: any = {
+	static flags: CreateFlags = {
 		...TransactionCreateCommand.flags,
 	};
 

--- a/commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/transaction/create.ts
+++ b/commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/transaction/create.ts
@@ -4,7 +4,7 @@ import { Application, PartialApplicationConfig } from 'lisk-sdk';
 import { getApplication } from '../../app/app';
 
 export class CreateCommand extends TransactionCreateCommand {
-	static flags = {
+	static flags: any = {
 		...TransactionCreateCommand.flags,
 	};
 

--- a/commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/transaction/sign.ts
+++ b/commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/transaction/sign.ts
@@ -3,8 +3,10 @@ import { TransactionSignCommand } from 'lisk-commander';
 import { Application, PartialApplicationConfig } from 'lisk-sdk';
 import { getApplication } from '../../app/app';
 
+type SignFlags = typeof TransactionSignCommand.flags & { [key: string]: Record<string, unknown> };
+
 export class SignCommand extends TransactionSignCommand {
-	static flags: any = {
+	static flags: SignFlags = {
 		...TransactionSignCommand.flags,
 	};
 

--- a/commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/transaction/sign.ts
+++ b/commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/transaction/sign.ts
@@ -4,7 +4,7 @@ import { Application, PartialApplicationConfig } from 'lisk-sdk';
 import { getApplication } from '../../app/app';
 
 export class SignCommand extends TransactionSignCommand {
-	static flags = {
+	static flags: any = {
 		...TransactionSignCommand.flags,
 	};
 

